app/views/spree/shared/_aside.html.erb

```diff
@@ -1,3 +0,0 @@
-<aside class="aside">
```

Comment:
Why are we removing this?

Reply:
aside has been moved away from the layout. Now it is rendered in the product index and taxon show, just as a container for taxons and filters. But I don't really find useful to have it as a global optional place to put contents. The presence of a side column is merely a design choice and its placement can differ between pages.